### PR TITLE
couple of fixes

### PR DESCRIPTION
**Pull request by OriBenHur-Any**
* add validate to the docker remove phase in clean.sh
* fixed the add_local_repo logic
* add kernel pre-check to make sure we can install and output an informative massage to the user if we can't

### CODE REVIEW

**Comment by OriBenHur-Any** [comment]
@sashas-anyvision Done

File: clean.sh [hunk 1]
```diff
@@ -243,6 +243,17 @@ function purge_docker {
   fi
 }
 
+function validate_docker_remove()
+{
+  if [ -x "$(command -v docker)" ]; then
```
Comment (sashas-anyvision) [inline clean.sh]: what is $(command -v docker) ? where command get from ?

File: clean.sh [hunk 2]
```diff
@@ -307,6 +320,7 @@ while test $# -gt 0; do
     break
 done
 
+
```
Comment (sashas-anyvision) [inline clean.sh]: please remove empty line

File: install.sh [hunk 3]
```diff
@@ -800,6 +800,38 @@ function nvidia_drivers_installation() {
   fi
 }
 
+function vercomp () {
+    local mod=${1//-/.}
```
Comment (sashas-anyvision) [inline install.sh]: please add comment here ... of what are you doing

**Pull request merged**